docs/error-codes.md

```diff
@@ -266,4 +472,154 @@ the error code. For example:
 
 #### `IL2049`: Unrecognized internal attribute 'attribute'
 
-- The internal attribute name 'attribute' being used in the xml is not supported by the linker, check the spelling and the supported internal attributes.
\ No newline at end of file
+- The internal attribute name 'attribute' being used in the xml is not supported by the linker, check the spelling and the supported internal attributes.
+
+#### `IL2050` Trim analysis: Making a generic type instantiation from 'type' which has 'DynamicallyAccessedMembersAttribute' on some of its generic parameters. ILLink currently doesn't analyze type values for generic parameters when making a generic type instantiation via 'System.Type.MakeGenericType'
```

Comment:
Could you unify quotes around DynamicallyAccessedMembersAttribute and other names. It looks like there are for some reasons used with "`", "'" and with no quotes

Reply:
It looks like the warning message is missing the reason.

```suggestion
#### `IL2050` Making a generic type instantiation from 'type' which has 'DynamicallyAccessedMembersAttribute' on some of its generic parameters is not supported.
```

Reply:
I tried to unify it like this:
* In messages it should be single-quoted : 'DynamicallyAccessedAttribute' - just like any other reference to names from the code (all method names, type names and so on should also be single-quoted)
* In the documentation it should be backtick-quoted so that the docs show it formatted as code.
* There should be no double quotes anywhere

Reply:
As for the message - I don't think we should say "not supported" - it is supported in the code, it's just that linker doesn't analyze it.

Reply:
ok but I think the sentence needs tweaking as "Making a generic type instantiation from 'type' which has 'DynamicallyAccessedMembersAttribute' on some of its generic parameters" is weird